commands/version/command.js

```diff
@@ -42,6 +42,10 @@ exports.builder = (yargs, composed) => {
       describe: "Specify cross-dependency version numbers exactly rather than with a caret (^).",
       type: "boolean",
     },
+    github: {
```

Comment:
Do you mind if we rename this `--github-release`? `--github` just doesn't "scope" the flag enough for me (in the context of `lerna version`, "github"... what?). It also makes the option a bit more self-descriptive in the implementation.

Reply:
Yup, of course.

Reply:
Thanks!